Ktisis/Interface/Windows/Toolbar/AdvancedWindow.cs

```diff
@@ -9,28 +9,22 @@
 
 namespace Ktisis.Interface.Windows.Toolbar {
 	public static class AdvancedWindow {
-		private static bool Visible = false;
-
-		public static Vector4 ColGreen = new(0, 255, 0, 255);
-		public static Vector4 ColRed = new(255, 0, 0, 255);
-
-		public static TransformTable Transform = new();
+		private static bool _visible;
```

Comment:
And renames this field

Reply:
Ah yeah I was working on refactoring on different local branch dunno how it came here ^^